app/routes/application.js

```diff
@@ -1,14 +1,6 @@
 import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
 
 export default class ApplicationRoute extends Route {
-
-  @service intl;
-
-  beforeModel() {
```

Comment:
I didn't realize keeping these lines would cause tests and Percy snapshots to always read French translations. My bad.